src/server/cmd/pachd/main.go

```diff
@@ -406,7 +406,8 @@ func doFullMode(config interface{}) (retErr error) {
 						blockAPIServer, err := pfs_server.NewBlockAPIServer(
 							env.StorageRoot,
 							0 /* = blockCacheBytes (disable cache) */, env.StorageBackend,
-							etcdAddress)
+							etcdAddress,
+							true /* duplicate */)
```

Comment:
Looks like this will solve #4081.

Reply:
Just updated the description of this PR so it will auto close that issue when it's merge. Obviously remove that if it's not true, but it seems like as part of this PR you've tested the expose object api flag with this.

Reply:
Sweet! I didn't know about the issue, but this should fix it. I am curious why an external user would be interested in it, but I posted that in the issue